packages/twenty-front/src/modules/object-record/record-show/components/FieldsCard.tsx

```diff
@@ -1,11 +1,5 @@
 import { RecordFieldList } from '@/object-record/record-field-list/components/RecordFieldList';
 import { useIsInRightDrawerOrThrow } from '@/ui/layout/right-drawer/contexts/RightDrawerContext';
-import styled from '@emotion/styled';
-
-const StyledPropertyBoxContainer = styled.div`
-  padding: ${({ theme }) => theme.spacing(3)};
```

Comment:
no more padding on FieldsCard level (as we want the separators to be full width). Instead we add it back on the children (SectionHeader, FieldList...)